broadinstitute/pearl
# [JN-487] admin left bar

This takes a first pass at implementing Erin's designs at https://dspuxteam.invisionapp.com/console/share/UQE7Z456RK2/986004503.  there are plenty of features still missing, but this seemed like the best self-contained section of work that makes the admin tool materially better.  There's a lot of trivial route refactoring and markup rearrangement, the major changes to pay attention to are:

1.  The overall navigation approach is shifting from portal-centric to study-centric.  So rather than the opening screen being select a portal, it's now selecting a study.
2.  There is now an environment selector and a study selector, which allows changing the study/environment shown without having to navigate up/down in the hierarchy first.  This means that we have to be much more careful about our useEffect dependencies  on page loads--if we're loading data that's study-specific, we need to make sure the useEffect triggering the load retriggers if the study is changed.  This is probably the most likely source of serious bugs in this PR, after misplaced links.
3. the new sidebar makes it possible to massively simplify a bunch of other pages, especially the participant list 
<img width="1428" alt="image" src="https://github.com/broadinstitute/juniper/assets/2800795/629d8ff6-0135-491b-9211-b954c18ace19">


TO TEST:
1.  populate 2+ portals in your local environment
2. go to localhost:3000
3. generally click around, exercise the left navbar 
4. when it appears, exercise the study and environment selectors, making sure data refreshes on selections
